javagen/src/main/java/com/azure/autorest/template/ServiceClientInterfaceTemplate.java

```diff
@@ -37,7 +37,7 @@ public final void write(ServiceClient serviceClient, JavaFile javaFile) {
 
         javaFile.javadocComment(comment ->
         {
-            comment.description(String.format("The interface for %1$s class.", serviceClient.getInterfaceName()));
+            comment.description(String.format("The interface for %1$s class.", serviceClient.getInterfaceName()), true);
```

Comment:
Another small inconsistency, this doesn't even have the ability to override the setting. And again, I'll say I like this always on with no ability to disable